routes/index.js

```diff
@@ -6,7 +6,7 @@ var router = express.Router();
 /* GET home page. */
 router.get('/', async function(req, res, next) {
 
-  const statusUrl = "http://localhost:8000/publishers/status";
+  const statusUrl = process.env.STATUS_DATA_URL || "http://localhost:8000/publishers/status";
```

Comment:
I was suggesting the Data URL would just be "http://localhost:8000/"

      const baseUrl = process.env.STATUS_DATA_URL || "http://localhost:8000";
      const statusUrl = baseUrl + "/publishers/status";

Or something.

That way if we have another end point later - maybe more details on a specific publisher at /publisher/details/:id or something - we can easily start using it without having to add more ENV variables or change existing ENV vars.

What do you think?

Reply:
I think that would be a good idea when that becomes the case ;)

Reply:
Personally I'd rather do it now and avoid us having to reconfigure live stuff later when it's this easy, but up to you, I'll approve and let you pick.